### DISCUSSION TITLE
Add installscript.vdf equivalent

### DISCUSSION THREAD

**Discussion by FanderWasTaken** [lutris/lutris#4891] [Ideas]
Hi. I know that Lutris install scripts install dependencies for games. However when there is no install script available, you're left with no dependencies automatically installed. I got tired of looking dependencies up online every time there's an issue with running a game, so [I made a script](https://github.com/FanderWasTaken/wine-dependency-hell-solver) to install all dependencies automatically. I was wondering if it was possible/viable to add this into Lutris. Maybe as a checkbox "Install dependencies" during install of a game?

**Comment by strycore** [comment]
I'm curious to know in which cases this has been helpful. Do you have examples of games?
The current directive for automatic installers is that if a game doesn't work with the default install then a script should be created.

I want to simplify that into just specifying what components are needed for a game.
Setting up a prefix with basically every common winetricks installed just seems overkill to me

**Reply by FanderWasTaken** [reply]
I have some examples: Darksiders (missing UI and cutscenes), The Darkness II (missing audio), and Mortal Kombat Komplete Edition (missing character select screen). Also, it was helpful when I was restoring games that were already installed on an older system. It could also be helpful in case of abandonware games, since some of them only come as pre-installed biniaries.

**Reply by strycore** [reply]
Abandonware games are very unlikely to use MF or recent versions of .NET. Those benefit more from being run in a 32bit prefix with possibly dgvoodoo2 enabled.
For the other games, I can see how this can be useful for testing. But in the end, I would rather have game dependencies for games being stored in our database. 
I'm thinking about creating a default prefix that is exclusive to Lutris (so we never default to ~/.wine). Maybe we can populate this prefix with what you  have in your script.
